sources/lib/api/gcp/stackdriver/monitoring/_timeseries.py

```diff
@@ -14,49 +14,54 @@
 
 """Provides access to metric data as pandas dataframes."""
 
+import gcloud.monitoring
 import pandas
 
 import gcp
-from . import _impl
 from . import _utils
 
 
-class Query(_impl.Query):
+class Query(gcloud.monitoring.Query):
```

Comment:
I thought we had discussed not building a dependency on gcloud in one of our earlier discussions, but if its being introduced despite that, hopefully it can remain an implementation detail?